habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -673,10 +686,29 @@ def _create_recep_info(
                     ],
                     axis=0,
                 )
-                receps[recep.name] = mn.Range3D(
+
+                rep_bound = mn.Range3D(
                     np.min(bounds, axis=0), np.max(bounds, axis=0)
                 )
+                receps[recep.name] = rep_bound
+
+                # Get the target rep id for this receptacle
+                obj = obj_mgr.get_object_by_handle(recep.parent_object_handle)
+                ao = ao_mgr.get_object_by_handle(recep.parent_object_handle)
+                if obj is not None:
+                    target_receps_id = (
+                        obj.object_id + self.habitat_config.object_ids_start
+                    )
+                elif ao is not None:
+                    target_receps_id = ao.visual_scene_nodes[0].semantic_id
```

Comment:
This seems wrong or arcane, can you explain? For objects it looks like the custom instance id for aos it grabs a pre-configured semantic id. :thinking: